Formula/dashing.rb

```diff
@@ -0,0 +1,36 @@
+class Dashing < Formula
+  desc "Generate Dash documentation from HTML files"
+  homepage "https://github.com/technosophos/dashing"
+  url "https://github.com/technosophos/dashing/archive/0.3.0.tar.gz"
+  sha256 "f6569f3df80c964c0482e7adc1450ea44532d8da887091d099ce42a908fc8136"
+
+  depends_on "glide" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    (buildpath/"src/github.com/technosophos/dashing").install buildpath.children
+    cd "src/github.com/technosophos/dashing" do
+      system "glide", "install"
+      system "go", "build", "-o", bin/"peco", "dashing.go"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    # download ruby docs just as dummy documentation to test with
+    system "curl", "-OL", "https://ruby-doc.org/downloads/ruby_2_5_0_core_rdocs.tgz"
```

Comment:
use a resource block to get this

Reply:
@ilovezfs How are you supposed to do that in a test only?

Reply:
Grep the formulae for `testpath.install` :)

Reply:
@ilovezfs I ended up using `resource.stage`, but thanks for pointing me in the right direction :)